src/components/Schedule.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import CSSModules from 'react-css-modules'
+import styles from 'assets/styles/schedule.styl'
+
+function Schedule(props) {
+  return (
+    <div className="container">
+      <div className="section six columns">
+        <h3>Today&rsquo;s Events</h3>
```

Comment:
If you want to use apostrophes, you can write:

```javascript
<h3>{ "Today's Events" }</h3>
```

This isn't a big deal, but in general if you're going to render text via jsx, it's better to use `{ 'some text' }` syntax over just the text. something something html artifacts.